cogs/hl.py

```diff
@@ -58,8 +69,10 @@ async def hl(self, ctx, *, code):
 
 		if len(code) > 2000:
 			raise commands.CommandError('Code contents too long to paste.')
-
-		message = await ctx.send(code)
+		try:
+			message = await ctx.send(content=code,reference=ctx.message.reference)
+		except HTTPException as e:
```

Comment:
```py
Traceback (most recent call last):
  ...
  File "/home/ace/test/AceBot/cogs/hl.py", line 74, in hl
    except HTTPException as e:
NameError: name 'HTTPException' is not defined
```

```suggestion
		except discord.HTTPException as e:
```
But really, don't send another message upon an HTTP Error. HTTP Error usually means something has gone wrong with the connection, not with any parameters.

Reply:
What this does is prevents an edge case if the message we try to reply to is deleted before we send the message.
Discord won't let the message go through, so we try to send it normally.

[docs](https://discordpy.readthedocs.io/en/latest/api.html?highlight=abc%20messageable%20send#discord.abc.Messageable.send)